src/Symfony/Component/Scheduler/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -0,0 +1,183 @@
+        $this->schemaSynchronizer = $schemaSynchronizer ?? new SingleDatabaseSynchronizer($this->driverConnection);
+        $this->autoSetup = $this->configuration['auto_setup'];
+    }
+
+    public function list(): array
+    {
+    }
+
+    public function get(string $taskName): ?array
+    {
+        $queryBuilder = $this->createQueryBuilder()->where('m.task_name = ?');
+        $data = $this->executeQuery($queryBuilder->getSQL(), [
+            $taskName,
+        ])->fetch();
+
+        return !$data ? null : $this->buildTask($data);
+    }
+
+    public function create(TaskInterface $task): void
+    {
+        $queryBuilder = $this->driverConnection->createQueryBuilder()
+            ->insert($this->configuration['table_name'])
+            ->values([
+                'task_name' => '?',
+                'expression' => '?',
+                'options' => '?',
+            ])
+        ;
+
+        $this->executeQuery($queryBuilder->getSQL(), [
+            $task->getName(),
+            $task->get('expression'),
+            $task->getOptions(),
+        ], [
+            Types::STRING,
+            Types::STRING,
+            Types::ARRAY,
+        ]);
+    }
+
+    public function pause(string $taskName): void
+    {
+    }
+
+    public function resume(string $taskName): void
+    {
+    }
+
+    public function removeAll(): void
+    {
+    }
+
+    public function delete(string $taskName): void
+    {
+        try {
+            $this->driverConnection->delete($this->configuration['table_name'], ['name' => $taskName]);
+        } catch (DBALException $exception) {
+            throw new TransportException($exception->getMessage(), 0, $exception);
+        }
+    }
+
+    public function setup(): void
+    {
+        $configuration = $this->driverConnection->getConfiguration();
+        // Since Doctrine 2.9 the getFilterSchemaAssetsExpression is deprecated
+        $hasFilterCallback = method_exists($configuration, 'getSchemaAssetsFilter');
+
+        if ($hasFilterCallback) {
+            $assetFilter = $this->driverConnection->getConfiguration()->getSchemaAssetsFilter();
+            $this->driverConnection->getConfiguration()->setSchemaAssetsFilter(null);
+        } else {
+            $assetFilter = $this->driverConnection->getConfiguration()->getFilterSchemaAssetsExpression();
+            $this->driverConnection->getConfiguration()->setFilterSchemaAssetsExpression(null);
+        }
+
+        $this->schemaSynchronizer->updateSchema($this->getSchema(), true);
+
+        if ($hasFilterCallback) {
+            $this->driverConnection->getConfiguration()->setSchemaAssetsFilter($assetFilter);
+        } else {
+            $this->driverConnection->getConfiguration()->setFilterSchemaAssetsExpression($assetFilter);
+        }
+
+        $this->autoSetup = false;
+    }
+
+    private function createQueryBuilder(): QueryBuilder
+    {
+        return $this->driverConnection->createQueryBuilder()
+            ->select('m.*')
```

Comment:
what is the m? maybe use task or schedule?

Reply:
Fixed